scripts/modal.js

```diff
@@ -1,3 +1,133 @@
+    name: 'Project name goes here',
+    demo: 'https://ksigei.github.io/portfolio2/',
+    source: 'https://github.com/ksigei/portfolio2',
+    langs: {
+      html: 'HTML/CSS',
+      ruby: 'Ruby on Rails',
+      js: 'JavaScript',
+    },
+  },
+  {
+    id: 'projects-wrapper',
+    name: 'Project name goes here',
+    demo: 'https://ksigei.github.io/portfolio2/',
+    source: 'https://github.com/ksigei/portfolio2',
+    langs: {
+      html: 'HTML/CSS',
+      ruby: 'Ruby on Rails',
+      js: 'JavaScript',
+    },
+  },
+  {
+    id: 'projects-wrapper',
+    name: 'Project name goes here',
+    demo: 'https://ksigei.github.io/portfolio2/',
+    source: 'https://github.com/ksigei/portfolio2',
+    langs: {
+      html: 'HTML/CSS',
+      ruby: 'Ruby on Rails',
+      js: 'JavaScript',
+    },
+  },
+  {
+    id: 'projects-wrapper',
+    name: 'Project name goes here',
+    demo: 'https://ksigei.github.io/portfolio2/',
+    source: 'https://github.com/ksigei/portfolio2',
+    langs: {
+      html: 'HTML/CSS',
+      ruby: 'Ruby on Rails',
+      js: 'JavaScript',
+    },
+  },
+];
+
+for (let i = 0; i <= 6; i += 1) {
+  htmlElements
+   += ` 
+   <div class="modal" id="modal2" data-animation="mixInAnimations">
+   <div class="modal-dialog">
+     <header class="modal-header">
+       <button class="close-modal" aria-label="close modal" data-close>
+         ✕  
+       </button>
+     </header>
+     <section class="modal-content">
+       <h3>Project name goes here</h3> 
+       <div class="list-languages">
+           <ul>
+               <li class="langs-li-dsk cursor"><p>${projects[projects.length - 1].html}</p></li>
+               <li class="langs-li-dsk cursor"><p>${projects[projects.length - 1].ruby}</p></li>
+               <li class="langs-li-dsk cursor"><p>${projects[projects.length - 1].js}</p></li>
+           </ul>
+       </div>
+       <div class="modal-carousel">
+           <img src="./images/carousel.svg" alt="">
+       </div>
+       
+       <div class="modal-description">
+           <p>
+               Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi Ut aliquip ex ea commodo consequat.  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.
+           </p>
+
+       </div>
+       
+       <div class="modal-btns">
+           <div class="btn-1">
+               <a href="#">
+                   <p>See Live <span><img src="./images/resume.png" alt="image"/></span></p>
+               </a>
+           </div>
+           <div class="btn-1">
+               <a href="#">
+                   <p>See Source <span><img src="./images/github.png" alt="image"/></span></p>
+               </a>
+           </div>
+       </div>
```

Comment:
- [ ] Kindly add the second icon, kindly make your buttons look like the original design by adding the right margin and padding and add the the button effects  so it looks like the original design 👍🏿 

 Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![btn](https://user-images.githubusercontent.com/69638013/157830263-12f357a9-80b3-44cf-9f25-0621a3b8ec55.png)  |  ![Screenshot_2](https://user-images.githubusercontent.com/69638013/157830232-33b81add-25a8-4dcb-99df-203ec69dad67.png)